proteus/cmeshToolsModule.cpp

```diff
@@ -1728,12 +1743,12 @@ static PyObject* SparsityInfo_getCSR(SparsityInfo *self,
 //     }
   int nnz=0;
   int dim[1];
-  dim[0] = int(self->columnIndecesMap.size())+1;
+  dim[0] = self->columnIndecesMap.size()+1;
```

Comment:
Reinserted the casts and assert statements in a0626121e540dbe69a3712b126ccadde16628012.